client/common/cmdline.c

```diff
@@ -4399,6 +4400,12 @@ static int freerdp_client_settings_parse_command_line_arguments_int(
 			if (!freerdp_settings_set_bool(settings, FreeRDP_RestrictedAdminModeRequired, enable))
 				return fail_at(arg, COMMAND_LINE_ERROR);
 		}
+		CommandLineSwitchCase(arg, "remoteGuard")
+		{
+			if (!freerdp_settings_set_bool(settings, FreeRDP_RemoteCredentialGuard, TRUE) ||
+			    !freerdp_settings_set_bool(settings, FreeRDP_ExtSecurity, TRUE))
+				return COMMAND_LINE_ERROR;
```

Comment:
use `fail_at(arg, COMMAND_LINE_ERROR)` to give useful feedback about where parsing failed